src/pages/_app.tsx

```diff
@@ -7,15 +7,15 @@ import { WalletModalProvider } from "@demox-labs/aleo-wallet-adapter-reactui";
 import { WalletProvider } from "@demox-labs/aleo-wallet-adapter-react";
 import Head from "next/head.js";
 
+import "~/styles/globals.css";
 import { api } from "~/utils/api.js";
 import { ThemeProvider } from "~/styles/index.js";
 import { APP_NAME } from "~/constants/index.js";
 import { useRouter } from "next/router.js";
 import { pages } from "~/router.js";
 import { AppContainer } from "~/containers/app-container/index.js";
 import { SoundProvider } from "~/containers/sound-provider/index.js";
-
-import "~/styles/globals.css";
+import { GameLoader } from "~/containers/game-loader";
```

Comment:
Extension, but actually just move the stuff from this component directly to the `SoundProvider`